### PR TITLE
Minor changes 

### PR DESCRIPTION
**Pull request by StefaniaGrimaldi**
[LISFLOOD_USERGUIDE_SuggestedEdits.pdf](https://github.com/ec-jrc/lisflood-code/files/5117381/LISFLOOD_USERGUIDE_SuggestedEdits.pdf)

### CODE REVIEW

**Review by domeniconappo — APPROVED** [review]

File: 3_step4_preparing-input-files/index.md [hunk 1]
```diff
@@ -21,30 +21,28 @@ All maps that are needed to run the model are listed in the table of Annex 12.
...
 
 ```xml
 <setoption choice="1" name="gridSizeUserDefined" \>
 ```
 
-LISFLOOD settings files and the use of options are explained in detail in **Chapter XXXX** of this document.
+LISFLOOD settings files and the use of options are explained in detail in a [dedicated chapter](https://ec-jrc.github.io/lisflood-code/3_step3_preparing-setting-file/) and [aanex](https://ec-jrc.github.io/lisflood-code/4_annex_settings_and_options/) of this document.
 
-#### Naming of meteorological variable maps
+**#### Naming of meteorological variable maps .... SHOULD WE REFER TO THE NETCDF CHAPTER? IS THE NETCDF CHAPTER A COMPLEMENT OR A REPLACEMENT TO THIS SECTION??**
```
Comment (domeniconappo) [inline 3_step4_preparing-input-files/index.md:45]: @StefaniaGrimaldi I'd avoid to put direct questions like this into docs. Remove the comment and then let's discuss between us. If further changes are required we can do it later.

**Review by domeniconappo — CHANGES_REQUESTED** [review]
Be sure that there aren't questions/comments (that are meant for internal use) in documentation.

File: 3_step4_preparing-input-files/index.md [hunk 2]
```diff
@@ -166,12 +164,12 @@ It is up to the user how the input data are organised. However, it is advised to
...
 The following Figure illustrates this:
 
-![](https://ec-jrc.github.io/lisflood_manual/media/image36.png){width="5.802083333333333in"
+![](https://github.com/StefaniaGrimaldi/lisflood-code/tree/gh-pages/media/image36.png){width="5.802083333333333in"
 height="4.541666666666667in"}
 
 ***Figure:*** *Suggested file structure for LISFLOOD*
 
-### Generating input base maps
+### Generating input base maps **CHECK THIS LAST SENTENCE**
```
Comment (domeniconappo) [inline 3_step4_preparing-input-files/index.md:172]: remove *CHECK etc.* please

File: 3_step4_preparing-input-files/index.md (hunk 1 above)
Comment (StefaniaGrimaldi) [reply 3_step4_preparing-input-files/index.md]: OK! Thanks!

File: 3_step4_preparing-input-files/index.md (hunk 2 above)
Comment (StefaniaGrimaldi) [reply 3_step4_preparing-input-files/index.md]: OK!

**Pull request merged**